### PR TITLE
chaos(engine): fix etcd-endpoints for servermaster

### PR DESCRIPTION
**Pull request by gozssky**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflow/issues/6511

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by gozssky** [comment]
/hold

Wait https://github.com/gozssky/tiflow/actions/runs/2753060449 to pass.

File: .github/workflows/dataflow_engine_chaos.yaml
Comment (gozssky) [inline .github/workflows/dataflow_engine_chaos.yaml:5]: For testing temporarily, it will be removed before merging.

**Review by CharlesCheung96 — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Comment by gozssky** [comment]
Test passed. See https://github.com/gozssky/tiflow/runs/7559346443.

**Pull request merged**
